uniqueidentifier_nullable.go

```diff
@@ -0,0 +1,37 @@
+package mssql
+
+import (
+	"database/sql/driver"
+)
+
+type NullableUniqueIdentifier UniqueIdentifier
```

Comment:
>ullableUniqueIdentifie [](http://example.com/codeflow?start=6&length=22)

It might be more idiomatic with the nullable types from the sql package (NullInt64, NullString, etc) to use the same pattern.

```go
type NullUniqueIdentifier
{
    UniqueIdentifier UniqueIdentifier
    Valid bool
}
```

Reply:
eab3da4af646247c1a4dcc98ca02cb494751ecdd

Reply:
what about a `Valid bool` instead of relying on an array of zeroes? The empty guid might be a valid non-null value for some applications.

Reply:
ccd7616aa7f43414a854fd0c59148f60a2b967cb